src/courses/mappers/03.md

```diff
@@ -0,0 +1,195 @@
+
+2) <i>**Controls**</i>: Controls are security standards that are used to prevent, mitigate, and address various security risks to sensitive information and infrastructure. In the case of OHDF, the <i>controls</i> structure is a collection of controls tested for by an external security service to ensure that the target complies with vulnerability and weakness prevention standards. Any given <i>profile</i> contains any number of <i>controls</i> which were tested against the target system during the original security service analysis.
+
+3) <i>**Results**</i>: The <i>results</i> structure contains information on the results of specific tests ran by the security service on the scan target against a security control. These results will always correlate to a certain control and will either report `passed` or `failed` to indicate the test status (other statuses exist but are rare) which cumulatively influence the determined compliance level of the scan target for some set of controls. Any given <i>control</i> contains any number of <i>results</i> which reflect the implemented tests to check if the target system is actually compliant with the control.
+
+## Breaking Down the Formal OHDF Schema
+
+::: note Full OHDF Schema
+The following section contains a breakdown of a streamlined version of the OHDF schema. For the full and up-to-date version, see [here](https://saf.mitre.org/framework/normalize/ohdf-schema).
+:::
+
+We can break down the formal OHDF schema in the same manner as before by observing each structure in the hierarchy from a top-down view.
+
+1) <i>**Profiles**</i>:
+```
+profiles: [
+  0: {
+    name             // Name of profile, usually the original security service tool; should be unique
+    version          // Version of security service tool
+    sha256           // Checksum of the profile; NOTE: AUTOMATICALLY GENERATED BY HDF CONVERTERS, DO NOT POPULATE
+    title            // Title of security service scan; should be human readable
+    maintainer       // Maintainer
+    summary          // Summary of security service export (e.g., the STIG header)
+    license          // Copyright license
+    copyright        // Copyright holder
+    copyright_email  // Copyright holder's email
+    supports         // Supported platform targets
+    attributes       // Inputs/attributes used in scan
+    groups           // Set of descriptions for the control groups (e.g., control IDs)
+    controls         // Controls substructure (see below)
+    status           // Status of profile (typically 'loaded')
+  }
+  ... // More items may exist if the security service produces multiple profiles per export
+]
+```
```

Comment:
It might be nice to include syntax highlighting for all the code blocks

Reply:
Good call